includes/widgets/social-icons.php

```diff
@@ -384,10 +384,23 @@ protected function register_controls() {
 			[
 				'label' => esc_html__( 'Size', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', '%', 'em', 'rem' ],
+				'default' => [
+					'unit' => 'px',
+					'size' => 14,
+				],
```

Comment:
New defaults values?

Reply:
@rodolphebertozzo This is probably the reason why the tests fail.

Reply:
Done @rami-elementor